core/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexRequest.java

```diff
@@ -440,6 +443,28 @@ public CreateIndexRequest updateAllTypes(boolean updateAllTypes) {
         return this;
     }
 
+    public int waitOnActiveShards() {
```

Comment:
`CreateIndexRequest` does not extend `ReplicationRequest`, nor does `UpdateRequest`, etc.